TangemSdk/TangemSdk/Common/CardSession.swift

```diff
@@ -68,12 +78,19 @@ public class CardSession {
     ///   - initialMessage: A custom description that shows at the beginning of the NFC session. If nil, default message will be used
     ///   - cardReader: NFC-reader implementation
     ///   - viewDelegate: viewDelegate implementation
-    public init(environment: SessionEnvironment, cardId: String? = nil, initialMessage: Message? = nil, cardReader: CardReader, viewDelegate: SessionViewDelegate) {
+    ///   - jsonCore: JSONRPCCore
+    public init(environment: SessionEnvironment,
+                cardId: String? = nil,
+                initialMessage: Message? = nil,
+                cardReader: CardReader,
+                viewDelegate: SessionViewDelegate,
+                jsonCore: JSONRPCCore) {
```

Comment:
Каждое следующее название хуже педыдущего :)
Его задача преобразовать JSON в Runnable. Тогда может так:
```suggestion
                jsonTransformer: JSONRPCTransformer) {
```